src/migrations/043_userinfo_oidc.ts

```diff
@@ -0,0 +1,56 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+
+  const result = await knex('changelog').select('id').where({ id: 43 });
+
+  if (result.length) {
+    // Migration has been applied using the old patch system
+    return;
+  }
+
+  await knex('changelog').insert({
+    id: 43,
+    timestamp: Math.floor(Date.now() / 1000)
+  });
```

Comment:
All this stuff is for the oldest migrations, before Knex was used. These (double-digit) old migrations live in here so there's an ugrade path.

This migration should use the timestamp format so it happens at the very end.